### ISSUE TITLE
Prettier 

### ISSUE THREAD

**Issue by stephenbe** [twbs/bootstrap#22919]
Is there any plan to use https://github.com/prettier/prettier for code formatting in the next version of bootstrap? Could really help for any contributions

**Comment by bkdotcom** [comment]
In what capacity?
Documentation?

Code syntax highlighting is out of scope for bootstrap.
I'm a fan of [prismjs](http://prismjs.com/) myself

**Comment by patrickhlauke** [comment]
Yeah I think this is outside of the scope of Bootstrap I'd say. We don't do automagic code highlighting, and if authors want it it's easy enough to integrate.

**Comment by stephenbe** [comment]
@patrickhlauke I would like you to reconsider this. This is not code highlighting. What it does is automatically  format your code for you. Major project slike react and  babel are using it. This will greatly improved code style cobsisty across all devs that work on Js, scss

**Comment by patrickhlauke** [comment]
ah i see. was lacking context here.

**Comment by stephenbe** [comment]
Yes I realized that I didn't add any info. 

Thank you for reopening :)

**Comment by FezVrasta** [comment]
If it helps decide, Popper.js already makes use of Prettier and it works flawlessly

**Comment by XhmikosR** [comment]
If anyone makes a PR, please CC me.

**Comment by lipis** [comment]
I made one #24323
